UnitTests/windows/TestEatHook.cpp

```diff
@@ -4,44 +4,45 @@
 #include "polyhook2/Tests/StackCanary.hpp"
 #include "polyhook2/Tests/TestEffectTracker.hpp"
 #include "polyhook2/PolyHookOsIncludes.hpp"
+#include "polyhook2/Detour/ADetour.hpp"
 
 EffectTracker eatEffectTracker;
 
 typedef void(* tEatTestExport)();
 uint64_t oEatTestExport;
 
-extern "C" __declspec(dllexport) NOINLINE void EatTestExport()
+extern "C" [[maybe_unused]] __declspec(dllexport) NOINLINE void EatTestExport()
```

Comment:
Please remove this function attribute I do not prefer to have these regardless of warnings.

Reply:
Sure